### ISSUE TITLE
@crustio/type-definitions v0.0.6 - Unknown Type warning on Rocky testnet 

### ISSUE THREAD

**Issue by RoyTimes** [crustio/crust.js#14]
It should be a minor thing that breaks nothing. 

Mainnet does not display such warning. 

I assume it's because of the testnet has deleted (or added) some functionalities from the mainnet and the `@crustio/type-definitions` package is configured to fit the mainnet env. 

To reproduce the error:
```javascript
const chain_ws_url = "wss://rocky-api.crust.network"
  const api = new ApiPromise({
    provider: new WsProvider(chain_ws_url),
    typesBundle: typesBundleForPolkadot,
  });

  await api.isReady
  const health = await api.rpc.system.health()
  console.log(health.toHuman())
```

Console displays the following warning. The RPC call runs well. 
`METADATA: Unknown types found, no types for CSMLedger, EraBenefits, FeeReductionBenefit`

**Comment by yuhui1208** [comment]
already fixed in this pr #13

**Comment by RoyTimes** [comment]
Nice. Closing
